docs/baking.md

```diff
@@ -0,0 +1,175 @@
+   -->
+# Baking with tezos-packaging on Ubuntu
+
+Tezos-packaging provides an easy way to install and set up infrastructure for
+interacting with Tezos blockchain.
+
+This article provides a step-by-step guide for setting up baking instance for Tezos on Ubuntu.
+
+## Prerequisites
+
+### Installing required packages
+
+In order to run a baking instance, you'll need the following Tezos binaries:
+`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
+
+Currently used proto is `008-PtEdo2Zk` (used on mainnet and edo2net) .
+Also, note that the corresponding ubuntu packages have protocol
+suffix in lowercase, e.g. list of available baker packages can be found
+[here](https://launchpad.net/~serokell/+archive/ubuntu/tezos/+packages?field.name_filter=tezos-baker&field.status_filter=published).
+
+To install them run the following commands:
+```
+# Add PPA with Tezos binaries
+sudo add-apt-repository ppa:serokell/tezos
+sudo apt-get update
+# Install packages
+sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+```
+
+Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
+systemd units for running the corresponding binaries in the background.
+
+## Setting up the tezos-node
+
+The easiest way to set up a node running on the `mainnet` or on one of the
+testnets (e.g. `edo2net`) is to use one of the predefined
+`tezos-node-<network>.services` systemd services provided in the `tezos-node`
+package.
+
+However, by default, these services will start to bootstrap the node from scratch,
+which will take a significant amount of time.
+In order to avoid this, here we suggest bootstrapping from a snapshot instead.
+
+### Setting up node service
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as its RPC address by default.
+
+In case you want to use different data directory or RPC address,
+you should update the service configuration. To edit the service configuration run
+```
+sudo systemctl edit --full tezos-node-<network>.service
+```
+
+### Bootstrapping the node
+
+In order to run a baker locally, you'll need a fully-synced local `tezos-node`.
+
+The fastest way to bootstrap the node is to import a snapshot.
+Snapshots can be downloaded from the following websites:
+* https://snapshots.tulip.tools/#/
+* https://snapshots-tezos.giganode.io/
+* https://xtz-shots.io/
+
+All commands within the service are run under the `tezos` user.
+
+Download the snapshot for desired network. We recommend to use rolling snapshot, this is
+the smallest and the fastest mode that is sufficient for baking (you can read more about other
+`tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
```

Comment:
Testing this on a fresh VM it seems there are permission problems here for the `tezos` user.

Can you check that's the case?

Reply:
Additionally, while we fix this I think that in the next release we should add aliases for `tezos-node-<network>` that are just `tezos-node` with the correct `data-dir`.

This would provide a great UX as these aliases names would match the service names as well.

Reply:
Ok so #148 is blocked, let's not wait for it for this article.